chocolatey/plugins/modules/win_chocolatey_feature.py

```diff
@@ -51,6 +58,12 @@
   win_chocolatey_feature:
     name: stopOnFirstPackageFailure
     state: enabled
+
+- name: Set Multiple Chocolatey Features
+  win_chocolatey_feature:
+    features:
```

Comment:
As the state is binary, the way I've seen this done elsewhere is you pass a list of features, and they are all given the same state.

Reply:
That's fair. I did wonder about using bools, e.g. `true` for enabled and `false` for disabled, but went pretty directly with what was currently implemented with the `state` parameter for my initial "do we like this?"